### DISCUSSION TITLE
在 FormTab 中自定义组件使用 Field 新增字段，表单验证时无法统计该报错（数量）

### DISCUSSION THREAD

**Discussion by tpc-ht** [alibaba/formily#3928] [Q&A 1.x] [answered]
![image](https://github.com/alibaba/formily/assets/59469126/bdf717fd-e519-4b0d-9d84-3c7d2fcc336c)

**Comment by tpc-ht** [comment]
请问有什么好的解决办法？
![image](https://github.com/alibaba/formily/assets/59469126/109e9b10-0017-4f62-944c-8b0301bdf6c6)

**Reply by beita-wang** [reply, marked as answer]
可以手动设置field.selfErrors 试一试 ，我设置这个属性后就会加入到统计数量中。

**Reply by tpc-ht** [reply]
好的。谢谢大佬
